packages/vuetify/src/components/VNoSsr/VNoSsr.tsx

```diff
@@ -0,0 +1,31 @@
+// Composables
+import { makeTagProps } from '@/composables/tag'
+
+// Utilities
+import { onMounted, ref } from 'vue'
+import { defineComponent } from '@/util'
+
+export default defineComponent({
+  name: 'VNoSsr',
+
+  props: {
+    ...makeTagProps({ tag: 'div' }),
+  },
+
+  setup (props, { attrs, slots }) {
+    const show = ref(false)
+
+    onMounted(() => {
+      show.value = true
+    })
+
+    return () => {
+      return show.value && (
```

Comment:
Does this have to return a root tag or can it be optional? Would like @KaelWD to chime in as well please.

Reply:
don't think so

Reply:
it doesn't need a root element though, just return whatever's in the default slot

Reply:
Now returning just the slot